manim/mobject/number_line.py

```diff
@@ -210,7 +210,18 @@ def get_tick_range(self):
             x_max = self.x_max
         else:
             x_max = self.x_max + 1e-6
-        return np.arange(self.x_min, x_max, self.x_step)
+
+        if self.x_min < x_max < 0 or self.x_max > self.x_min > 0:
+            return np.arange(self.x_min, x_max, self.x_step)
+
+        start_point = 0
```

Comment:
When 0 is not a value on the number line, the improper tick placement persists:
![image](https://user-images.githubusercontent.com/70682032/120595070-d14d7500-c40f-11eb-87b4-813f88cc4d2c.png)

I think this might be influenced by this line.

Reply:
Could you share your code to reproduce this please? When I do

`      
n = NumberLine((3, 9), include_numbers=True)

self.add(n)
`

I get:

![AxesExample_ManimCE_v0 6 0](https://user-images.githubusercontent.com/32387857/120702959-9d745d00-c4ac-11eb-8f23-235c1c411d8f.png)

Reply:
```py
class Num(Scene):
    def construct(self):
        first = NumberLine(
            x_range=[PI, 3 * PI],
            include_numbers=True,
            include_tip=False,
        )
        self.add(first)
```

Reply:
It seems that this is controlled by `decimal_number_config` so if I change your example to:

```
class Num(Scene):
    def construct(self):
        first = NumberLine(
            x_range=[PI, 3 * PI],
            include_numbers=True,
            include_tip=False,
            decimal_number_config={'num_decimal_places': 2}
        )
        self.add(first)
```

Then I would get:

![AxesExample_ManimCE_v0 6 0](https://user-images.githubusercontent.com/32387857/120888373-e54fcd00-c5ef-11eb-8cae-554277a0cbb3.png)

So if the step size is 1 then I think the result makes sense? It doesn't necessarily need to hit the max value. I think to do that we would need an alternative to step size such as 'number_of_steps' and calculate the steps between the two values? But that is maybe outside the scope of this PR.

You can change the step size though to get something closer to the min and max such as making step size PI / 3 which results in:

![AxesExample_ManimCE_v0 6 0](https://user-images.githubusercontent.com/32387857/120888531-a3735680-c5f0-11eb-8f55-0a76cc4710fc.png)

Reply:
Ah I see, I misinterpreted the un-centered ticks as being unintentional, so I brought it up just in case. I'm fine with it being like this and if someone wants centered tick placement for `NumberLines` that do not include zero then it can be added later.